attestation-service/docs/policy.md

```diff
@@ -37,6 +37,14 @@ If the extension is registered in the EAR specification, the key will be positiv
 Otherwise, the key should be negative.
 The extension value can be any valid JSON, including nested objects.
 
+The default policy includes the extension `ear.trustee.identifiers`, which is used
```

Comment:
Let's mention that in CoCo case where agent policy is also enabled, the extension exists.